base/applications/notepad/lang/he-IL.rc

```diff
@@ -157,15 +157,18 @@ BEGIN
     STRING_TEXT_FILES_TXT "קבצי טקסט (*.txt)"
     STRING_TOOLARGE "File '%s' is too large for notepad.\n\
 Please use a different editor."
-    STRING_NOTEXT "You didn't enter any text. \
-\nPlease type something and try again"
-    STRING_DOESNOTEXIST "File '%s'\ndoes not exist\n\n\
-Do you want to create a new file ?"
-    STRING_NOTSAVED "File '%s'\nhas been modified\n\n\
-Would you like to save the changes ?"
+    STRING_TOOLARGE "הקובץ '%s' גדול מידי עבור פנקס הרשימות\n\
```

Comment:
`STRING_TOOLARGE` is duplicated (you translated it, but didn't removed original english string)